### DISCUSSION TITLE
Same FaceSwap flow on comfyUI

### DISCUSSION THREAD

**Discussion by bsormagec** [lllyasviel/Fooocus#1309] [Q&A]
Hello everyone, 

After researching various models and software, I have found that the best face changer is the one that can keep the natural look of the face. Can someone please assist me in finding this application or recommend something similar that I can use on my ComfyUI? Thank you.

**Comment by amirshachar** [comment]
I have the same question. I particularly like the face that Fooocus also maintains the hairstyle. I don't know what's going on behind the scenes but it would be great if someone knows how to build this workflow manually in ComfyUI. Reactor custom node seem to only do the face.
